### ISSUE TITLE
Percentage border radius values do not work

### ISSUE THREAD

**Issue by chrisinajar** [niklasvh/html2canvas#354]
Appears to be at the fault of this particular line:
https://github.com/niklasvh/html2canvas/blob/master/src/Core.js#L264
That'll parse in values like "50%" as the int "50", which is interpreted as pixels.

This is my fast and loose fix, please adapt properly if you have the time:

``` javascript
_html2canvas.Util.getCSS = function (element, attribute, index) {
    if (previousElement !== element) {
      computedCSS = document.defaultView.getComputedStyle(element, null);
    }

    var value = computedCSS[attribute];

    if (/^background(Size|Position)$/.test(attribute)) {
        return parseBackgroundSizePosition(value, element, attribute, index);
    } else if (/border(Top|Bottom)(Left|Right)Radius/.test(attribute)) {
      var arr = value.split(" "),
          attrDirection;

      if (arr.length <= 1) {
          arr[1] = arr[0];
      }

      // first value handling for % based values
      if (/[0-9]+%/.exec(arr[0])) {
        attrDirection = attribute.split(/[A-Z]/);
        switch (attrDirection[1]) {
          case "op": // Top
          case "ottom": // Bottom
            arr[0] = asInt(computedCSS['width']) / (100 / asInt(arr[0]));
            break;
          case "eft": // Left
          case "ight": // Right
            arr[0] = asInt(computedCSS['height']) / (100 / asInt(arr[0]));
            break;
        }
      }

      // second value handling for % based values
      if (/[0-9]+%/.exec(arr[1])) {
        attrDirection = attribute.split(/[A-Z]/);
        switch (attrDirection[2]) {
          case "op": // Top
          case "ottom": // Bottom
            arr[1] = asInt(computedCSS['width']) / (100 / asInt(arr[1]));
            break;
          case "eft": // Left
          case "ight": // Right
            arr[1] = asInt(computedCSS['height']) / (100 / asInt(arr[1]));
            break;
        }
      }
      return arr.map(asInt);
    }

  return value;
};
```

This was literally researched, found, and fixed in like 20 minutes, so please excuse any level of derpness observed (also please let me know since I'm using this code in prod :beers: )

**Comment by Ajaxy** [comment]
Reproducing in 0.5.0-alpha1

**Comment by niklasvh** [comment]
Fixed in 1.0.0
